src/OrchardCore.Modules/OrchardCore.Contents/Startup.cs

```diff
@@ -42,6 +44,13 @@ public class Startup : StartupBase
     {
         public override void ConfigureServices(IServiceCollection services)
         {
+            // Re-register core shapes to not be tied to the application's module,
+            // so that that they can be overridden by the current theme.
+            services.ReplaceShapeAttributes<CoreShapes>();
```

Comment:
This is not a valid solution. These shapes are part of the DisplayManagement project, which can be used with Contents. Think about a SaaS solution for instance. The solution would be to add them to a fake feature, or a display management feature, that would be low priority. Even if it means to change the `AddShapeAttribute` to accept an extra parameter to mention it need to use a default feature. Note that this feature could be a static variable one, like `FeatureInfo.Default` that could be used when necessary.

Reply:
Yes, the idea was that if `OC.Contents` is not enabled, these core shapes are still registered through `DisplayManagement`.

> The solution would be to add them to a fake feature

I thought about this but it needs more code, e.g a type can be tied to a feature if it is registered through a startup. Through the application we already have helpers to do pseudo startups but there are all tied to the application's module, we don't have any helper yet that would tie a given pseudo startup to a fake feature.

But i could think about it if it is worth to work on it, let me know.